### DISCUSSION TITLE
General discussion thread!!

### DISCUSSION THREAD

**Discussion by setraj** [arvpo/blazer#37] [General]
Creating this thread for general discussion around this project.

**Comment by arvpo** [comment]
btw, found a new issue : https://github.com/arvyshka/blazer/issues/38

**Comment by arvpo** [comment]
btw found a link to test your last PR, works as expected

https://filesamples.com/samples/video/flv/sample_3840x2160.flv

**Reply by setraj** [reply]
we may not use a third-party open URL directly in our tests. Our tests may start failing tomorrow if that URL is taken down.
we need to find another way to simulate a local file server for our integration tests.
for unit tests, we can mock the server response.

**Reply by arvpo** [reply]
Yes, you are right. I was looking for some resource in internet, to test last PR manually and came across this website. well, yes we need to have our own test system which can handle these cases

**Comment by arvpo** [comment]
https://filesamples.com/ has lot of various sample files to downloading in clients
